### DISCUSSION TITLE
Setting Subcatchment & Conduits Parameters Through API

### DISCUSSION THREAD

**Discussion by HosseinVT** [pyswmm/pyswmm#459] [Q&A] [answered]
I am currently working on implementing automatic calibration for the SWMM model, which involves adjusting various parameters such as the Manning coefficient and conduits roughness. I am curious to know if it is possible to access and adjust these values through the API.

**Comment by karosc** [comment, marked as answer]
@HosseinVT, this is not yet possible using pyswmm. swmmio might be able to do what you want by allowing you create multiple inp files programmatically, then you can run those simulations using pyswmm.

**Comment by HosseinVT** [comment]
Thank you !
